src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -496,13 +496,18 @@ void generateOverallSummaryFiles(
 
   // write CastVoteRecords for all contests to the provided folder
   // returns a list of files written
-  List<String> writeGenericCvrCsv(List<CastVoteRecord> castVoteRecords,
+  List<String> writeGenericCvrCsv(
+      List<CastVoteRecord> castVoteRecords,
       Collection<Contest> contests,
-      String csvOutputFolder)
-      throws IOException {
+      String csvOutputFolder,
+      String contestId
+  ) throws IOException {
     List<String> filesWritten = new ArrayList<>();
     try {
       for (Contest contest : contests) {
+        if (!isNullOrBlank(contestId) && !contest.getId().equals(contestId)) {
```

Comment:
I don't understand the idea behind this and would feel more comfortable if @moldover reviewed as well.

Reply:
Oh, it's just that the Dominion source may contain multiple contests, but when we're doing a tabulation, we only care about one of them (indicated by contestId), so I'm skipping all the others.

Reply:
@tarheel This is correct logic, but it does point out a (easy to fix) problem: the DominionCvrReader should be filtering on contest Id when the CastVoteRecords are first read.  Then, no additional filtering is needed, either here, or during tabulation.  Do you want me to fix that in your branch?  Or it's simple enough to add around line 237 where we are already validating the contest id is defined in the 'contests' file.